go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/obscuronet/go-obscuro
 go 1.20
 
 require (
+	github.com/allegro/bigcache v1.2.1
```

Comment:
are both versions of bigcache required?

- github.com/allegro/bigcache v1.2.1
- github.com/allegro/bigcache/v3 v3.1.0

Reply:
I didn't change anything that would impact this, I think this was the IDE arranging the imports.

Ended up doing go mod tidy and it placed the bigcache/ as an indirect import.